models/issues/issue_project.go

```diff
@@ -14,27 +14,24 @@ import (
 
 // LoadProject load the project the issue was assigned to
 func (issue *Issue) LoadProject(ctx context.Context) (err error) {
-	if issue.Project == nil {
-		var p project_model.Project
-		has, err := db.GetEngine(ctx).Table("project").
+	if issue.Projects == nil {
+		err = db.GetEngine(ctx).Table("project").
 			Join("INNER", "project_issue", "project.id=project_issue.project_id").
-			Where("project_issue.issue_id = ?", issue.ID).Get(&p)
-		if err != nil {
-			return err
-		} else if has {
-			issue.Project = &p
-		}
+			Where("project_issue.issue_id = ?", issue.ID).OrderBy("title").
```

Comment:
Why we need to order the result here?
And maybe it is better to convert the function name into `LoadProjects`?

Reply:
Again, this is just a holdover from @tyroneyeh 's original thing. All I did was try to resolve the conflicts.

Reply:
I'm sorry. I was just reviewing this PR and didn't notice your comment above.
Are you planning to continually improve this PR or just resolve the conflicts?